page-map.js

```diff
@@ -46,6 +46,11 @@ const pageList = [
             path: 'decent/quickstart',
           },
           {
+            name: 'Architectures',
```

Comment:
Let's call it 'Architecture' (singular). Plural will look weird in TOC.

Reply:
updated directly on master, prematurely merged this.